### PR TITLE
mecab: build with system GCC

### PR DESCRIPTION
**Pull request by danielnachun**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/linuxbrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/linuxbrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
- [ ] Have you included the output of `brew gist-logs <formula>` of the build failure if your PR fixes a build failure. Please quote the exact error message.

-----

### CODE REVIEW

**Comment by danielnachun** [comment]
Do I need to revision bump this and rebuild the bottle now that we can build it with the system GCC?

**Comment by iMichka** [comment]
Yes, please. I think it's better to ship a new bottle; even if its only a build-time dependency.

**Review by iMichka — APPROVED** [review]

File: Formula/mecab.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class Mecab < Formula
   # Canonical url is https://drive.google.com/uc?export=download&id=0B4y35FiV1wh7cENtOXlicTFaRUE
   url "https://deb.debian.org/debian/pool/main/m/mecab/mecab_0.996.orig.tar.gz"
   sha256 "e073325783135b72e666145c781bb48fada583d5224fb2490fb6c1403ba69c59"
+  revision 1
```
Comment (iMichka) [inline Formula/mecab.rb:7]: revision 1 unless OS.mac?

**Review by iMichka — CHANGES_REQUESTED** [review]
`revision 1 unless OS.mac?`

**Pull request merged**